charts/mattermost-operator/templates/mysql-operator/statefulset.yaml

```diff
@@ -0,0 +1,99 @@
+      app: {{ .Values.mysqlOperator.appName }}-operator
+      release: {{ .Values.mysqlOperator.appName }}-operator
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.mysqlOperator.appName }}-operator
+        release: {{ .Values.mysqlOperator.appName }}-operator
+{{- if .Values.mysqlOperator.statefulSet.annotations }}
+      annotations:
+        {{- toYaml .Values.mysqlOperator.statefulSet.annotations | nindent 8 }}
+{{- end }}
+    spec:
+      serviceAccountName: {{ .Values.mysqlOperator.appName }}-operator
+      containers:
+        - name: operator
+          image: "{{ .Values.mysqlOperator.image.repository }}:{{ .Values.mysqlOperator.image.tag }}"
+          imagePullPolicy: "{{ .Values.mysqlOperator.image.pullPolicy }}"
+          env:
+            - name: ORC_TOPOLOGY_USER
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.mysqlOperator.appName }}-operator-orc
+                  key: TOPOLOGY_USER
+            - name: ORC_TOPOLOGY_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.mysqlOperator.appName }}-operator-orc
+                  key: TOPOLOGY_PASSWORD
+          args:
+            - --leader-election-namespace={{ .Values.mysqlOperator.appName }}-operator
+            # connect to orchestrator on localhost
+            - --orchestrator-uri=http://127.0.0.1:3000/api
+            - --sidecar-image={{ .Values.mysqlOperator.sidecar.image.repository }}:{{ .Values.mysqlOperator.sidecar.image.tag }}
+        - name: orchestrator
+          image: "{{ .Values.mysqlOperator.orchestrator.image.repository }}:{{ .Values.mysqlOperator.orchestrator.image.tag }}"
+          imagePullPolicy: "{{ .Values.mysqlOperator.orchestrator.image.pullPolicy }}"
+          ports:
+            - containerPort: 3000
+              name: web
+              protocol: TCP
+            - containerPort: 10008
+              name: raft
+              protocol: TCP
+          env:
+            - name: POD_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.podIP
+          envFrom:
+            - prefix: ORC_
+              secretRef:
+                name: {{ .Values.mysqlOperator.appName }}-operator-orc
+          volumeMounts:
+            - name: data
+              mountPath: /var/lib/orchestrator/
+            - name: config
+              mountPath: /templates/
+          livenessProbe:
+            timeoutSeconds: 10
+            initialDelaySeconds: 200
+            httpGet:
+              path: /api/lb-check
+              port: 3000
+          # https://github.com/github/orchestrator/blob/master/docs/raft.md#proxy-healthy-raft-nodes
+          readinessProbe:
+            timeoutSeconds: 10
+            httpGet:
+              path: /api/raft-health
+              port: 3000
+      volumes:
+        - name: config
+          configMap:
+            name: {{ .Values.mysqlOperator.appName }}-operator-orc
```

Comment:
all these overall can be in helper tpl so you can have one place with the preparation.